### PR TITLE
Tweak Contributing docs to help people run the tests

### PR DESCRIPTION
**Pull request by aviflax**
(I hope this markup is kosher; I don’t know restructured text.)

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -54,11 +54,17 @@ Then start the ``compose`` session::
 
 At this point, you should be able to see the Web UI at http://localhost:8000
 
-Once your session is up and running, you can make yourself a superuser account::
+Once your session is up and running, you can::
+
```
Comment (shuuji3) [inline docs/contributing.rst:58]:
```suggestion
Once your session is up and running, you can:

```

Comment (shuuji3) [reply docs/contributing.rst:58]: I noticed one warning in the build log: https://github.com/jointakahe/takahe/actions/runs/3613790002/jobs/6089780689#step:5:28

This is due to the double colon syntax `::` called literal blocks. Only we need to put two colons when placing a code block in the next paragraph. So you don't have to place two here. 🙂 

ref. [reStructuredText Primer — Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks)

**Review by shuuji3 — CHANGES_REQUESTED** [review]

**Comment by shuuji3** [comment]
You can also build documentation with the following commands:

```shell
cd takahe/
pip install -r docs/requirements.txt
make docs
```

This outputs HTML files under the `docs/_build/html/` directory.

File: docs/contributing.rst (hunk 1 above)
Comment (shuuji3) [reply docs/contributing.rst:58]: You can build documentation with the following commands:

```shell
cd takahe/
pip install -r docs/requirements.txt
make docs
```

This outputs HTML files under the `docs/_build/html/` directory.

Comment (shuuji3) [reply docs/contributing.rst]: Looks good! 👍🏻

**Review by shuuji3 — APPROVED** [review]

**Comment by andrewgodwin** [comment]
Thanks!

**Pull request merged**